src/commands/start.command.ts

```diff
@@ -6,12 +6,26 @@ import { createServer, Socket } from "net";
 
 import { ScriptDefinition } from "../script-definition.type";
 
-export const start = async (pmConfigFilePathOverride?: string): Promise<void> => {
+export const start = async (pmConfigFilePathOverride?: string, options?: { only?: string[] }): Promise<void> => {
     const pmConfigFilePath = pmConfigFilePathOverride ? pmConfigFilePathOverride : "dev-pm.config.js";
     const { scripts }: { scripts: ScriptDefinition[] } = await import(`${process.cwd()}/${pmConfigFilePath}`);
     const processes: { [key: string]: ChildProcess } = {};
     const logSockets: { socket: Socket; name: string | null }[] = [];
     let shuttingDown = false;
+    let scriptsToStart: ScriptDefinition[] = [];
+
+    if (options?.only && options?.only?.length > 0) {
+        options.only.map((scriptName) => {
+            const foundScript = scripts.find((script) => script.name === scriptName);
+            if (foundScript) {
+                scriptsToStart.push(foundScript);
+            } else {
+                console.log(`${colors.bgRed.bold.black(" dev-pm ")} Script ${scriptName} not found in dev-pm config`);
```

Comment:
We probably should use `console.error` here. In my opinion, we might even abort execution completely.

Reply:
Good idea
I switched to `console.error` and abort execution using `process.exit(1);`